### ISSUE TITLE
Chisels and Bits Rendering Issue

### ISSUE THREAD

**Issue by devalidating** [ValkyrienWarfare/Valkyrien-Warfare-Revamped#38]
After assembling the ship, some Chisels and Bits blocks fail to render properly. 
Top image is before, Bottom is after.
![2017-02-01_21 09 30](https://cloud.githubusercontent.com/assets/19363057/22538686/2270262a-e8c9-11e6-80aa-770880e3472e.png)
![2017-02-01_21 09 36](https://cloud.githubusercontent.com/assets/19363057/22538687/227122c8-e8c9-11e6-9371-d4480e337164.png)

**Comment by BigBastard** [comment]
**Update:** From the Minecraft Forums, I found an extended description I figured I'd add here for future refrence.

_"I figured out that the chisel and bits pieces would become visible if the ship was reloaded, either by reloading the world or going to the front and moving it out of loaded chunks and then going back, only to disappear again when any blocks or added or removed near them (and by near I mean in about 30 blocks of them). It also seems to be related to the complexity of the bit, since the fences stayed the whole time, and the stairs, but slopes and stuff didn't. I think its a memory issue."_

I'll try getting this fixed on Friday/Saturday; if what this description says is correct, then it won't take much time until I get this figured out.
